### DISCUSSION TITLE
Визуальный редактор для маршрутов.

### DISCUSSION THREAD

**Discussion by it-artur** [mikopbx/Core#57] [Ideas (Идеи)]
Ребят, просьба добавить. Визуальный редактор для маршрутов это очень облегчает. работу.
У вас на старых версиях она была, потом его убрали.
![InkedcsJbr2p5_LI](https://user-images.githubusercontent.com/6401807/103014260-8883bd00-454f-11eb-9a8d-b783187f3154.jpg)
![6197fc0d0e888d56b29d4745c07d4976](https://user-images.githubusercontent.com/6401807/103014367-b832c500-454f-11eb-93c2-e18cc05f13e4.png)

**Comment by jorikfon** [comment]
@artsaroyan А первый скриншот это какая система?

**Comment by it-artur** [comment]
ВАТС от мсн.ру
Именно реализация редактора очень хорошая упрощает жизнь))

**Comment by jorikfon** [comment]
Это большая задача, она у нас в планах, скорее всего будет дополнительный модуль для MikoPBX
